libraries/AP_HAL_ChibiOS/hwdef/scripts/chibios_hwdef.py

```diff
@@ -762,21 +762,14 @@ def get_flash_pages_sizes():
         return [ 128 ] * (get_config('FLASH_SIZE_KB', type=int)//128)
     elif mcu_series.startswith('STM32F100') or mcu_series.startswith('STM32F103'):
         return [ 1 ] * get_config('FLASH_SIZE_KB', type=int)
+    elif mcu_series.startswith('STM32L4') and mcu_type.startswith('STM32L4R'):
+        # STM32L4PLUS
```

Comment:
@tridge It is STM32L4P.

Reply:
the mcu_type comes from this line: ```
MCU STM32L4xx STM32L4R5xx
```
is that a typo? Did you mean it to be L4P? If so then the py file needs changing too

Reply:
Yeah ok @tridge i forgot